EncodingStreamingHighAvailability/HighAvailability.JobOutputStatus/Startup.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+
+[assembly: FunctionsStartup(typeof(HighAvailability.JobOutputStatus.Startup))]
+
+namespace HighAvailability.JobOutputStatus
+{
+    using Azure.Storage.Queues;
+    using HighAvailability.AzureStorage.Services;
+    using HighAvailability.Interfaces;
+    using HighAvailability.Services;
+    using Microsoft.Azure.Cosmos.Table;
+    using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+    using Microsoft.Extensions.DependencyInjection;
+    using System;
+
+    /// <summary>
+    /// Implements startup logic for job output status Azure function.
+    /// See for more details about dependency injection for Azure Functions
+    /// https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection
```

Comment:
Can you also list the required variables and settings related? Maybe like to final doc page

Reply:
Will add link to final document.